requirements.yml

```diff
@@ -5,6 +5,7 @@ collections:
   - name: prometheus.prometheus
     version: 0.4.0
   - name: grafana.grafana
+    version: 10.3.1
```

Comment:
Doesn't look like a `grafana-ansible-collection` release

Reply:
ok I found the correct version : 2.2.5